dbfit-java/build.gradle

```diff
@@ -294,6 +308,80 @@ project('netezza') {
         compile project(':dbfit-java:core')
         runtime 'org.netezza:nzjdbc3:7.2@jar'
     }
+
+    task connectionPropertyFilename {
+        ext.propertiesFile = 'TestDbConnectionDbFitNetezza.properties'
```

Comment:
That looks like constant. What do you think about just declaring it as `ext.connectionPropertyFilename = ...`?